### PR TITLE
Allow generation of AWW Performance report at state and district level

### PR DESCRIPTION
**Pull request by dmydlo**
Hi @calellowitz,
I have created the AWW Performance report at state and district level.
JIRA [ticket](https://dimagi-dev.atlassian.net/browse/ICDS-337).
Need QA: Yes
Environment: n/a
Locally Tested: Yes
Regards, Dawid

### CODE REVIEW

**Comment by dmydlo** [comment]
Hi @calellowitz, error in the tests is random and not related to the changes.

**Comment by calellowitz** [comment]
@dmydlo 
This doesnt appear to be behind a feature flag. It should be for testing.

**Comment by dmydlo** [comment]
> @dmydlo
> This doesnt appear to be behind a feature flag. It should be for testing.

@calellowitz 
I have updated that.

File: custom/icds_reports/utils/aggregation_helpers/aww_incentive.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ def aggregation_query(self):
         ) (
           SELECT
             %(state_id)s AS state_id,
+            awcm.district_id,
```
Comment (calellowitz) [inline custom/icds_reports/utils/aggregation_helpers/aww_incentive.py]: @dmydlo 
You mention you tested this locally but this this query looks wrong (you add a column to the `SELECT` but not the `INSERT` clause. Are you sure this query works the way that is intended?

**Pull request merged**